src/main/java/frc/robot/constants/Constants.java

```diff
@@ -15,4 +15,5 @@ public final class Constants {
   public static final DriveConstants drive = new DriveConstants();
   public static final AutoConstants auto = new AutoConstants();
   public static final FalconConstants falcon = new FalconConstants();
+  public static final IntakeConstants intake = new IntakeConstants();
```

Comment:
Do not make an instance of your constants. Import frc/robot/constants/IntakeConstants into your code.